### PR TITLE
printed blank lines before book list in show books function, added 2 …

### PR DESCRIPTION
**Pull request by qdaniel4**
added blank lines before book list in show books function
@sf7293gv  please review

### CODE REVIEW

**Comment by sf7293gv** [comment]
Good and simple. Ready to merge.

**Comment by claraj** [comment]
@sf7293gv @qdaniel4 I left some comments on this PR - please resolve. You'll need a new branch and new PR.

File: ui.py [hunk 1]
```diff
@@ -25,12 +25,14 @@ def show_books(books):
      :param books: the book list """
 
     # TODO print a blank line before and after the book list or no books message
-
+    print() #blank line
```
Comment (claraj) [inline ui.py:28]: This comment isn't necessary

File: ui.py [hunk 2]
```diff
@@ -25,12 +25,14 @@ def show_books(books):
...
     if books:
         for book in books:
             print(book)
     else:
+        print() #blank line
```
Comment (claraj) [inline ui.py:33]: Same

File: ui.py [hunk 3]
```diff
@@ -25,12 +25,14 @@ def show_books(books):
...
         print('No books to display')
+    print() #blank line
```
Comment (claraj) [inline ui.py:35]: Same

File: ui.py [hunk 4]
```diff
@@ -25,12 +25,14 @@ def show_books(books):
      :param books: the book list """
 
     # TODO print a blank line before and after the book list or no books message
```
Comment (claraj) [inline ui.py:27]: You've fixed this issue, so remove the word "TODO" in the comment

**Pull request merged**